documents/Strategies/Building strategy.tex

```diff
@@ -0,0 +1,38 @@
+\documentclass{article}
+
+\usepackage{graphicx}
+
+\author{Michiel van den Berg}
+\title{Strategy for building buildings}
+
+\begin{document}
+\pagenumbering{gobble}
+
+\maketitle{}
+\newpage{}
+
+\section{Introduction}
+
+In this document the strategy about the decision of building buildings and which buildings will be explained. In the first section the reason to build a building will be defined, and in the second section the choice for a particular building will be clarified. This document will be used to implement the following user story:\\
+\\
+\textit{"As an agent\\
+When in a simulation\\
+I am able to build stuff if I think 
+this is benificial for me\\
+To optimize a certain indicator."\\}
+
+\section{When/why to build buildings}
+In this section the reasons to build a building will be explained. The first reason to build a building, is that it can influence an indicator in a positive way. Another reason why to build something, is that a building can give you profit in terms of your budget.
+
+
+\subsection{Direct influence}
+There is one direct reason to build a building. We created an indicator that has the goal to make high buildings. The more high buildings, the better the indicator will be. Another indicator we have, is the extension of education space. So to achieve this we need to build more education building. The agent will build stuff as long as the budget holds and as long as it gives a positive influence on the indicators.
+
+
+\section{Which building to build}
+This section explains which building to build. Because a part of our vision is to build high buildings, the buildings that are build, needs to be high buildings. In the Tygron engine, high buildings are of the type “Luxe variant”, so these buildings needs to be built until the indicator has reached 100\%. When the indicator has reached 100\%, it does not matter which buildings are built for the high building indicator. For the extension of the education space, it does not matter which building is built, as long as the building is the most beneficial for the indicator. 
+
```

Comment:
Keep in mind that an indicator probably won't reach 100% in a regular simulation run.